### PR TITLE
Deprecate the repo

### PR DESCRIPTION
**Pull request by liangxia**
@jhou1 @wsun1 @xltian  Could you help merge the PR, then move the repo to readyonly ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,3 @@
 # v3-testfiles
-The repo to store the test files (json/yaml/scripts) which will be used during the OpenShift v3 tests.
+The repo was deprecated. Integrate your test data with your scripts. That is to say, if your script exist in verification-tests, put your test files in folder testdata of the repo verification-tests; if your script exist in cucushift, put your test files in folder testdata of the repo cucushift.
```
Comment (jhou1) [inline README.md]: was deprecated -> is deprecated

Comment (liangxia) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,3 @@
 # v3-testfiles
-The repo to store the test files (json/yaml/scripts) which will be used during the OpenShift v3 tests.
+The repo is deprecated. Integrate your test data with your scripts. That is to say, if your script exist in verification-tests, put your test files in folder testdata of the repo verification-tests; if your script exist in cucushift, put your test files in folder testdata of the repo cucushift.
```
Comment (pruan-rht) [inline README.md]: exist -> exists

Comment (liangxia) [reply README.md]: Fixed.

**Pull request merged**